storage/types.go

```diff
@@ -33,19 +33,61 @@ type Node struct {
 }
 
 // NodeID uniquely identifies a Node within a versioned MerkleTree.
```

Comment:
It seems like "versioned" is irrelevant here.

Reply:
Probably true, removed.